docs/resources/library.md

```diff
@@ -92,26 +93,24 @@ resource "databricks_library" "fbprophet" {
 
 Installing Python libraries listed in the `requirements.txt` file.  Only Workspace paths and Unity Catalog Volumes paths are supported.  Requires a cluster with DBR 15.0+.
 
-
 ```hcl
 resource "databricks_library" "libraries" {
   cluster_id   = databricks_cluster.this.id
   requirements = "/Workspace/path/to/requirements.txt"
 }
 ```
 
-
 ## Python EGG
 
 ```hcl
-resource "databricks_dbfs_file" "app" {
+resource "databricks_file" "app" {
```

Comment:
According to the docs: https://docs.databricks.com/aws/en/libraries/#python-library-support

> Installing Python egg files is only supported on Databricks Runtime 13.3 LTS and below, and only for dedicated or no isolation shared access modes. In addition, **you cannot install Python egg files on volumes or workspace files**. Use Python wheel files or install packages from PyPI instead.

Reply:
We also need to add deprecation message here